apps/web/src/api-clients/cost-analysis/budget/composables/use-budget-usage-api.ts

```diff
@@ -0,0 +1,17 @@
+import { SpaceConnector } from '@cloudforet/core-lib/space-connector';
+
+import type { ListResponse } from '@/api-clients/_common/schema/api-verbs/list';
+import type { BudgetUsageListParameters } from '@/api-clients/cost-analysis/budget-usage/schema/api-verbs/list';
+import type { BudgetUsageModel } from '@/api-clients/cost-analysis/budget-usage/schema/model';
+
+
+export const useBudgetUsageApi = () => {
+    const actions = {
+        list: SpaceConnector.clientV2.costAnalysis.budgetUsage.list<BudgetUsageListParameters, ListResponse<BudgetUsageModel>>,
+        analyze: SpaceConnector.clientV2.costAnalysis.budgetUsage.analyze,
```

Comment:
Suggestion: there is `.analyze` verb's response type!
```suggestion
        analyze: SpaceConnector.clientV2.costAnalysis.budgetUsage.analyze<BudgetUsageAnalyzeParameters, AnalyzeResponse<BudgetUsageAnalyzeResult>>,
```